net.fasterland.converseen.json

```diff
@@ -0,0 +1,57 @@
+{
+    "id": "net.fasterland.converseen",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "converseen",
+    "rename-desktop-file": "converseen.desktop",
+    "rename-appdata-file": "converseen.appdata.xml",
+    "finish-args": [
+        "--device=dri",
+        "--filesystem=home",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland"
+    ],
+    "modules": [
+        {
+          "name": "imagemagick",
+          "config-opts": [
+            "--disable-static"
+          ],
+          "sources": [
+            {
+              "type": "archive",
+              "url": "https://github.com/ImageMagick/ImageMagick/archive/7.0.11-6.tar.gz",
+              "sha256": "8adc1605784653b078572b825e8cd1d3d54f8a1b4ba86b32ca253c038f7e4c37"
+            }
+          ]
+        },
+        {
+            "name": "converseen",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/Faster3ck/Converseen.git",
+                    "tag": "v0.9.9.5",
+                    "commit": "856cef30d3b0ef7d7582db0dc569a248fceb118e"
+                },
+                {
+                    "type": "shell",
+                    "commands": [ "sed -i 's#/usr/share/metainfo/#${CMAKE_INSTALL_PREFIX}/share/metainfo/#' CMakeLists.txt" ]
```

Comment:
make sure you submit a patch upstream for that.

Reply:
See https://github.com/Faster3ck/Converseen/pull/45 which was ignored and now can't be applied.